esque/cli/commands.py

```diff
@@ -382,39 +446,52 @@ def get_consumergroups(state: State, output_format: str):
 @output_format_option
 @default_options
 def get_topics(state: State, prefix: str, output_format: str):
+    """List all topics."""
     topics = state.cluster.topic_controller.list_topics(search_string=prefix, get_topic_objects=False)
     topic_names = [topic.name for topic in topics]
     click.echo(format_output(topic_names, output_format))
 
 
-@esque.command("consume", help="Consume messages of a topic from one environment to a file or STDOUT")
+@esque.command("consume")
 @click.argument("topic", required=True)
-@click.option("-f", "--from", "from_context", help="Source Context", type=click.STRING, required=False)
-@click.option("-n", "--numbers", help="Number of messages", type=click.INT, default=sys.maxsize, required=False)
-@click.option("-m", "--match", help="Message filtering expression", type=click.STRING, required=False)
+@click.option(
+    "-f",
+    "--from",
+    "from_context",
+    metavar="<source_ctx>",
+    help="Source context. If not provided, the current context will be used.",
+    type=click.STRING,
+    required=False,
+)
+@click.option(
+    "-n", "--numbers", metavar="<n>", help="Number of messages.", type=click.INT, default=sys.maxsize, required=False
+)
+@click.option(
+    "-m",
+    "--match",
+    metavar="<filter_expresion>",
+    help="Message filtering expression.",
```

Comment:
I think the filter/match expressions might need some documentation, too. Maybe we can introduce another command? Like `esque help filter` ? And then refer the user to that.